web-api/src/persistence/postgres/trialSessions/createOrUpdateTrialSessionCases.ts

```diff
@@ -0,0 +1,35 @@
+import { settlePromises } from '@web-api/utilities/settlePromises';
+import { TCaseOrder } from '../../../../../shared/src/business/entities/trialSessions/TrialSession';
+import { pgInsertInto } from '../utils/operation/pgInsertInto';
+
+import { toKyselyNewTrialSessionCase } from './mapper';
+
+export const createOrUpdateTrialSessionCases = async ({
```

Comment:
Probably should just call this `upsertTrialSessionCases`.

Reply:
We _almost_ named it `upsertTrialSessionCases`